src/System.Net.Http/src/System/Net/Http/ByteArrayHelpers.cs

```diff
@@ -43,7 +43,7 @@ internal static bool EqualsOrdinalAsciiIgnoreCase(string left, ReadOnlySpan<byte
 
         internal static unsafe string GetStringFromByteSpan(ReadOnlySpan<byte> bytes)
         {
-            // TODO: Use new Span-based Encoding overload when available
+            // TODO #22431: Use new Span-based Encoding overload when available
```

Comment:
I found this because I was searching for all places we were using DangerousGetPinnableReference.  While it's not directly related, didn't seem worthwhile to submit a separate PR to add 7 characters to a comment :)